README.md

```diff
@@ -743,7 +743,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
   * [appneta.com](https://www.appneta.com/) — Free with 1-hour data retention
   * [appspector.com](https://appspector.com/) - Mission control, for remote iOS/Android/Flutter debugging. Free for small traffic usage (64MB of logs).
   * [assertible.com](https://assertible.com) — Automated API testing and monitoring. Free plans for teams and individuals.
-  * [blackfire.io](https://blackfire.io/) — Blackfire is the SaaS-delivered Application Performance Solution. Free Hacker plan (PHP only)
+  * [blackfire.io](https://blackfire.io/) — Blackfire is the SaaS-delivered Application Performance Solution. Free plan for [Students and Teachers](https://www.blackfire.io/students/)  only (PHP/Python only)
```

Comment:
Thanks. I think we should just remove it entirely then.

Reply:
@ripienaar done.